nautobot/docs/installation/migrating-from-postgresql.md

```diff
@@ -18,7 +18,7 @@ This will result in a file named `nautobot_dump.json`.
 
 ## Create the MySQL database
 
-Create the MySQL database for Nautobot, ensuring that it has the correct character set (`utf8mb4`) and collation (`utf8mb4_bin`) settings for case-sensitivity (MySQL will otherwise be case-insensitive by default, which may cause problems when importing your data from the case-sensitive PostgreSQL database dump). Refer to the instructions for [CentOS/RHEL](./centos.md) or [Ubuntu](./ubuntu.md) as necessary if you are unsure how to set up MySQL and create the Nautobot database.
+Create the MySQL database for Nautobot, ensuring that it has the correct character set (`utf8mb4`) and collation (`utf8mb4_general_ci`) settings for case-sensitivity (MySQL will otherwise be case-insensitive by default, which may cause problems when importing your data from the case-sensitive PostgreSQL database dump). Refer to the instructions for [CentOS/RHEL](./centos.md) or [Ubuntu](./ubuntu.md) as necessary if you are unsure how to set up MySQL and create the Nautobot database.
```

Comment:
This is the one point that gives me pause here - in a Postgres deployment, the database itself is case-sensitive (e.g. you can create devices "DEVICE1" and "device1" and they will coexist just fine) but case-insensitive queries are possible. It seems that in MySQL it's all or nothing - either the database and its queries are case-sensitive, or the database and its queries are case-insensitive. So I'm no longer certain that we should across-the-board recommend one or the other collation setting with MySQL - we should probably document the pros and cons of both options clearly and leave it up to the user to decide.

Reply:
Agreed. This sketches me out here. Why are these tests only failing in Docker containers, but not otherwise? I would like to better understand this.

Reply:
These docs will need to be revised to instead say that MySQL is case-insensitive by default, and because of this you will need to account for this in your database dumps PRIOR to migrating from PostgreSQL to MySQL. We may also consider offering some ways to determine this ahead of time?

Reply:
After a deep excursion into the depths of Django's lookup strategy in Postgres v MySQL, I do not think it's recommended to advise people create case sensitive MySQL databases. Django doesn't expect it at all and it's `i*` expressions are not built to handle it.

TLDR: For Postgres Django does essentially `UPPER(field) = UPPER(lookup)` for iexact. However for MySQL does `field LIKE lookup`. This is insufficient when MySQL is running a case sensitive collation. This seems to be a nastier tradeoff than cleaning up data on migrate. If you need case sensitivity, you should use Postgres.